calico-cloud/visibility/packetcapture.mdx

```diff
@@ -48,11 +48,37 @@ For a simple use case workflow see, [Faster troubleshooting of microservices, co
 
 ## How To
 
+- [Enable Packet capture](#enable-packet-capture)
 - [Packet capture in Service Graph](#packet-capture-in-service-graph)
 - [Packet capture using the command line](#packet-capture-using-the-command-line)
 - [Store and rotate capture files](#store-and-rotate-capture-files)
 - [Enforce RBAC for capture tasks for CLI users](#enforce-rbac-for-capture-tasks-for-cli-users)
 
+### Enable Packet capture
+
+##### Enable PacketCapture using kubectl
+
+Create a PacketCaptureAPI custom resource, named `tigera-secure` in the cluster.
```

Comment:
Do the names of CRs usually end with `API`?

Reply:
Nope, it was named PacketCaptureAPI because it's part of operator v1. To avoid collision with the existing PacketCapture resource in v3, we discussed with the operator maintainers and decided on this name."